lib/9pfs/9pfs_vnops.c

```diff
@@ -499,25 +509,31 @@ static int uk_9pfs_write(struct vnode *vp, struct uio *uio, int ioflag)
 	if (!uio->uio_resid)
 		return 0;
```

Comment:
It actually seems that there already is an identical check before the `uk_9p_walk`, at line 494. I will remove this one altogether, since it's redundant.